examples/acados_python/getting_started/pendulum_ode_gnsf_functions.json

```diff
@@ -0,0 +1,8 @@
+	"casadi_version": "3.5.1"
+}
```

Comment:
is this meant to be versioned?

Reply:
I just added the version to the json file to the dumped file.
I am not fully convinced serialization will be compatible across CasADi versions.